### DISCUSSION TITLE
OSPOlogyLive Apeldoorn Presentations

### DISCUSSION THREAD

**Discussion by geekygirldawn** [todogroup/ospology#437] [OSPO Discussions]
Creating a thread for the Apeldoorn presentations starting with my presentation:
* [Using CHAOSS Metrics Models for Contributing to and Consuming OSS](https://github.com/todogroup/ospology/files/14403346/CHAOSS_Contributing_Consuming_OSPOlogy.pdf)

**Comment by Ainali** [comment]
Here is my presentation for today:

* [Using the Standard for Public Code now and in the future](https://files.publiccode.net/nextcloud/index.php/s/OSPOlogy2024)

**Comment by ospo-user** [comment]
Here is my presentation from yesterday

- [OSPOlogy.live-2024-Navigating-and-growing-The-OSPO-adventure-for-the-Dutch-Tax-Administration.pdf](https://github.com/todogroup/ospology/files/14444960/OSPOlogy.live-2024-Navigating-and-growing-The-OSPO-adventure-for-the-Dutch-Tax-Administration.pdf)

**Comment by JohnnyCustom** [comment]
Here's my presentation from the round table on the first day. 
[Opspology.Live 2024 context for roundtable on expanding your Container tech capabilites with OSS ENG John.pptx](https://github.com/todogroup/ospology/files/14448471/Opspology.Live.2024.context.for.roundtable.on.expanding.your.Container.tech.capabilites.with.OSS.ENG.John.pptx)

**Comment by gruetter** [comment]
And here's my InnerSource presentation, as well
[innersource-ospology-apeldoorn.pdf](https://github.com/todogroup/ospology/files/14491689/innersource-ospology-apeldoorn.pdf)

**Comment by ElenaFdR** [comment]
Here's my presentation about what we've learned at the Foundation for Public Code:

* [Stewardship organizations and the public sector.pdf](https://files.publiccode.net/nextcloud/index.php/s/GNan9NH3m68SyQ7)
